### ISSUE TITLE
Member responsibilities – JAT, who likes to do what? 

### ISSUE THREAD

**Issue by armenos** [joomla/accessibility#15]
## Who likes to do what? 
For example 
I’ve asked Justyna to help me with organization and support me somehow.
But she can tell which areas she likes to be active, like organizing our meetings, etc.

I know Yair likes to develop, so we know he want to make extensions, like menu, video player, content fixer or a11y issue marker. We will discuss on this in another topic. 

Feel free to say what are you expertise fields and what you do like to do and what you could do.

Every member can be active to make new comers feeling welcome to our project, etc.

**Comment by uglyeoin** [comment]
I'm an expert in HTML Email coding.  But that's probably not going to be relevant here.  

I'm a native English speaker but I would not say it is my strongest skill.  I can probably help a little with spelling/grammar if you are not a native speaker/writer but I would most certainly not see myself as an expert.  Most English natives have better skills than I, and Ray is most certainly more knowledgeable.  

I have built some plugins/modules for Joomla! but they are quite basic.  I enjoy it though which is why I volunteered to write the plugin for the skip menu.

I joined the team because I think accessibility is important and I really want Joomla! to get to AA because I want us to be a front runner in that field.  I built a website for a client who works with disabilities and although most of my clients don't care about that I enjoyed doing it.  

I'm ok with CSS, know a small bit of PHP, HTML is ok, and JS is useless.  Good with Photoshop, good with InDesign so can make banners etc if need be.

Hope that helps.

**Comment by tarotray** [comment]
I have two major skills to offer: 1) I am pretty good at writing and proof-reading documentation, in English.
2) I am a full-stack Joomla developer. I write PHP, Javascript, HTML5 and CSS3.0. I am trained and experienced, to a certain extent, in Structured Query Language (MS SQL and mySQL). It is up to others to judge how good I am at all of these skills.

I have built modules, components and plug-ins for Joomla, although I am no great expert.

I am always happy to learn new skills, where time is available and I have been studying WCAG, ARIA, GIT, GitHub etc. since I joined the team. It's unfortunate that my time is somewhat limited but I can definitely commit to at least a few hours work/study per week.

My preference is to not make non-technical decisions or to take responsibilities for anything other than my coding work. So, it is always better to allocate me tasks and then I shall work on them until I achieve success. I am highly results-orientated and see myself as a mere humble coder, albeit with a great deal of experience and a low level of specialisation in any one coding skill.

**Comment by zwiastunsw** [comment]
1. I have a great knowledge of web accessibility.
2. I can test and evaluate the accessibility of websites.
3. I would like to focus on developing documentation and accessibility guides.
4. Unfortunately, I do not know English very well, so I need help and cooperation from other authors in this work.
5. I have a lot of experience in managing teamwork and managing distributed teams. I can help with teamwork, defining tasks, monitoring.

**Comment by chmst** [comment]
I have worked fo 30 years as a projectmanager for software projects and consultant for software quality - but this was ten years ago and the new techniques of agile develoment and github and so on are new to me too.

Since 6 yeras about I've built up knowledge in Joomla programming and am developer for Joomla components, modules, plugins. Sometimes I contribute to joomla core, but not often, in general I use to test the PRs of my colleagues.
I am good in Joomla programming - PHP and mySQL,  rather good in templating HTML/ CSS (Boostrap v4), hopeless in JS and JQuery (but learnig), noob in all server questions.

I am board member of an organisation whose main concern is the inclusion of handicaped (most of them paralysed). Making the website for them opened my eyes for web accessibility and gave me much experience in real-life - I learned how disabilites can exclude someone from using the web. I am nearly deaf and cannot understand the sreenreader - cannot participate in video conferences or hangouts.

I like reviewing code, testing code, writing and evaluating code, teaching, training and making presentations.
